front_end/src/actions/cityActions.js

```diff
@@ -0,0 +1,79 @@
+import axios from "axios";
+
+// Type imports
+import {
+  GET_CITY,
+  GET_CITY_WEATHER,
+  LOADING_CITY,
+  LOADING_CITY_WEATHER,
+  CITY_ERRORS
+} from "./types";
+
+// API key imports
+import { weatherMapAPIKey } from "../apiKeys/keys";
+
+// Fetches city data using the provided city id
+export let fetchCityById = cityId => dispatch => {
+  setLoadingCity();
+
+  // TODO: Uncomment and fill in missing code below once backend API is implemented then remove prototype code
```

Comment:
how far along is the back? would be good to merge the actual code... not a prototype and then needing to go back and change again